src/es/ikigaimangas/src/eu/kanade/tachiyomi/extension/es/ikigaimangas/IkigaiMangas.kt

```diff
@@ -40,7 +40,7 @@ class IkigaiMangas : HttpSource(), ConfigurableSource {
     override val supportsLatest: Boolean = true
 
     private val cookieInterceptor = CookieInterceptor(
-        baseUrl.substringAfter("://"),
+        "",
```

Comment:
domain?

Reply:
chapter url redirects to other domains, the interceptor check with `endsWith` so this apply the cookies on all domains